src/custom-element.js

```diff
@@ -803,9 +803,11 @@ function createBaseCustomElementClass(win) {
       this.isConnected_ = true;
 
       if (!this.everAttached) {
-        this.classList.add('i-amphtml-element');
-        this.classList.add('i-amphtml-notbuilt');
-        this.classList.add('amp-notbuilt');
+        this.classList.add(
```

Comment:
This breaks IE11 support: https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#Browser_compatibility. We could use `this.className += ' foo bar baz'`, but that'll cause a tokenization of all current classnames, too.

Reply:
We could introduce a transform for this so the module build can use `TokenList.add(...params)` but transpile to separate `TokenList.add(param)` for the legacy build.

Reply:
I'll polyfill instead.

Reply:
Let's make sure the polyfill is not added to the module build too.